sdk/webpubsub/Azure.Messaging.WebPubSub/src/Generated/WebPubSubServiceClient.cs

```diff
@@ -1581,12 +1581,12 @@ internal HttpMessage CreateSendToAllRequest(RequestContent content, ContentType
             request.Headers.Add("Accept", "application/json, text/json");
             request.Headers.Add("Content-Type", contentType.ToString());
             request.Content = content;
-            message.ResponseClassifier = ResponseClassifier202.Instance;
             return message;
         }
 
         internal HttpMessage CreateSendToAllRequest(RequestContent content, IEnumerable<string> excluded, RequestContext context)
         {
+            context.ResponseClassifier ??= ResponseClassifier204.Instance;
```

Comment:
You shouldn't mutate customer-provided instances.

Reply:
Gone.